force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1785,6 +1785,14 @@
         <shortDescription>Error when an attempt is made to change the Status of a Closed Elevate RD</shortDescription>
         <value>You can&apos;t edit the Status for an Elevate connected Recurring Donation when Status is Closed.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateRDCannotBeFixedLength</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when trying to change an Elevate RD to Recurring Type Fixed</shortDescription>
+        <value>An Elevate-connected Recurring Donation Cannot have a Recurring Type of Fixed.</value>
```

Comment:
@sam-knox New error for Elevate Recurring Donations to review when you have a chance!

Reply:
@lparrott - Can Elevate RDs be Fixed to begin with, or is it that you just can't change it from Open to Fixed. Or is only Open supported in Elevate?

Reply:
No, they can only be Open. When creating a new RD, the payment widget will hide if the user selects "Fixed". On Edit of an Elevate RD, the payment widget will still hide when the user selects "Fixed". Since that RD is already connected to Elevate, it'll fail the back-end validation in this scenario.

Reply:
Great, thanks Ben. So the message you have will work, with one slight change:

```suggestion
        <value>An Elevate-connected Recurring Donation cannot have a Recurring Type of Fixed.</value>
```

@lparrott